python/cuvs/cuvs/test/test_ivf_pq.py

```diff
@@ -166,3 +166,45 @@ def test_ivf_pq_search_params(params):
         lut_dtype=params["lut"],
         internal_distance_dtype=params["idd"],
     )
+
+
+def test_save_load():
```

Comment:
A significant portion of this code will be the same for this test for all indices, I wonder if we should refactor the common code?

Reply:
I've refactored to reduce the duplicate code in the last commit - we now have a `test_serialization.py` script that tests each of cagra/ivf_flat/ivf_pq using the same common function